custom_components/tuya_local/devices/alecoair_d16_dehumidifier.yaml

```diff
@@ -48,19 +48,19 @@ primary_entity:
     - id: 19
       name: error
       type: bitfield
-      # Docs mention E1 and E2, it isn't clear which is tank full, so for now
-      # treat both as that.
+      # E2 might be a compressor fault.
       mapping:
         - dps_val: 0
           value: OK
+          icon: "mdi:cup-outline"
         - dps_val: 1
-          value: E1
+          value: "Water Tank Full or Removed"
           icon: "mdi:cup-water"
-          icon_priority: 2
+          icon_priority: 1
```

Comment:
I think it is better to leave these as 2, so they do not clash with the power off icon.  If you feel strongly that the abnormal conditions should be indicated even when the switch is off, then change that to 2 so the priorities are clearly separate.